react/react1/week1/to-do/src/Components/ToDoList.jsx

```diff
@@ -0,0 +1,35 @@
+
+import ToDoItem from './ToDoItem'
+import ToDoList from './ToDoHeader'
+const listArray = [
+  {  task: "Get out of bed", deadLine: "Wed Sep 13 2017" },
+  { task: "Brush teeth", deadLine: "Thu Sep 14 2017" },
+  { task: "Eat breakfast", deadLine: "Fri Sep 15 2017" }
+];
+
+function ToDo() {
+  return (
+    <div className="header">
+      <ToDoList/>
```

Comment:
`TodoList.jsx` is actually a header, so would be interesting changing the name.

Reply:
Thanks for your time and review I will fix this